public/app/services/apps.ts

```diff
@@ -82,9 +82,10 @@ function removeDuplicateApps(app: App[]) {
   });
 }
 
-export async function fetchApps(): Promise<
-  Result<App[], RequestError | ZodError>
-> {
+export async function fetchApps(
+  start: number,
+  end: number
```

Comment:
Not all calls to `fetchApps` are passing these parameters in `apps.spec.ts`
- Do we want to consider making them optional args?
- Will settings the 
- Everywhere else in pyroscope oss, we have `from` and `until` instead of `start` and `end`. Do we want to keep the different terminology exclusive to the POST method body, or do we want it to propagate through the function signature and the variables in calling code.
  > ```  
  >   const start = formatAsOBject(state.continuous.from);
  >   const end = formatAsOBject(state.continuous.until);
  >   const startMs = start.getTime();
  >   const endMs = end.getTime();
  >    ```
- What affect will `start` and `end` being `0` have on the back end? What about `null` or `undefined`? Will they be treated equivalently?

Reply:
> Do we want to consider making them optional args?

That's a good idea! Fixed in https://github.com/grafana/pyroscope/pull/2482/commits/f597fe19be9d2b362b117011848f13d0945a16cf.

> Will settings the

🤔 

> Everywhere else in pyroscope oss, we have from and until instead of start and end. Do we want to keep the different terminology exclusive to the POST method body, or do we want it to propagate through the function signature and the variables in calling code.

Keeping `from` and `to` as conventions in the UI is probably best. I'll change references to `start` and `end`. Fixed in https://github.com/grafana/pyroscope/pull/2482/commits/4e932cc9c6edab2db9dae299819ec527dd1acfbe and https://github.com/grafana/pyroscope/pull/2482/commits/f597fe19be9d2b362b117011848f13d0945a16cf.

> What affect will start and end being 0 have on the back end? What about null or undefined? Will they be treated equivalently?

If either `start` or `end` are `0`, `null`, or `undefined`, then the backend will fallback to the legacy behavior of only querying the head block and not querying the block store.